packages/integrations/tailwind/README.md

```diff
@@ -84,9 +84,9 @@ If it isn't there, you add your own `tailwind.config.(js|cjs|mjs)` file to the r
 
 The Astro Tailwind integration handles the communication between Astro and Tailwind and it has its own options. Change these in the `astro.config.mjs` file (_not_ the Tailwind configuration file) which is where your project's integration settings live.
 
-#### config.path
+#### configFile
```

Comment:
```suggestion
#### config.path (deprecated in vxxxxx)

In vxxxxx, `config.path` was replaced with [configFile](#configFile). Please update to the latest version of `@astrojs/tailwind` and [update your integration config](add a section showing the code example from the file diff and link to it!) accordingly.

#### configFile (added in vxxxx; replaces `config.path`)
```

What about something *like* this here? Not removing the field entirely but showing deprecation, and for which version?

Then, we'll make a section in the README itself for updating, exactly like you showed in the changeset, that can be used as reference.

I think leaving this in for a while is helpful enough to anyone troubleshooting, looking at the OTHER updated sections' code examples and asking themselves, "Wait, what the heck is `configFile`? I have `config.path`?? and then they come see this here and it's easy to see that it's a 1:1 swap of names.

Reply:
> If they're upgrading a major, they should check the changelog instead.

I feel like you're expecting a lot of users here. 😅 And remember, MOST people will see this page *in docs*, where they won't have easy/direct access to the changelog.

The above shorter is fine, BUT I would add something with a link to the changelog then that says something like:

> See the changelog for updating your config

? What about that? Then, whatever's decided here, do the same below? I think with a link to follow where they can see the example in the changelog is OK. I might later add something myself in the way of an example of the updated config, but as long as there's a clear path to that changelog, that works so this can be non-blocking!

Reply:
A link to the changelog works for me!

I think eventually everyone has to learn to find the changelog when updating majors. Not all package has docs explaining migrating X to Y. Maybe I'm a bit harsh to people 😄 

Anyways I give the changelog suggestion a go first.